src/components/media-player/media-player-track.js

```diff
@@ -207,6 +207,9 @@ function MediaPlayerTrack({
     { 'is-hidden': screenMode === MEDIA_SCREEN_MODE.overlay }
   );
 
+  const isBufferingWhilePlaying = trackPlayState.current === MEDIA_PLAY_STATE.buffering && trackPlayState.beforeBuffering === MEDIA_PLAY_STATE.playing;
+  const shouldPlay = trackPlayState.current === MEDIA_PLAY_STATE.playing || isBufferingWhilePlaying;
```

Comment:
Great find and nice solution!

Reply:
Thanks :-) Let's hope we don't have to find much more stuff in the future